### DISCUSSION TITLE
depressions_info.csv file??

### DISCUSSION THREAD

**Discussion by reecejk** [opengeos/lidar#24] [Q&A]
Hi, I was able to install the ArcGIS Pro Lidar package to extract sinks and depression hierarchies. However, I do not have an output depressions_info.csv file. Is this no longer an output for the latest lidar package or am I overlooking it somehow? I am utilizing bathymetry with negative elevations, so maybe that's a problem for the depressions_info.csv output file? I'm not sure if the previous reported issue may have caused a halt for this .csv output, but the information within the depressions_info.csv file (Table 1 in the supporting lidar documentation) is a central theme to a chapter of my PhD dissertation and I hope to publish and cite your work within the next year. I'm very thankful for your contributions so far!

**Comment by giswqs** [comment]
Did you try out the example data? 

```python
import os
import pkg_resources
from lidar import *

# identify the sample data directory of the package
package_name = 'lidar'
data_dir = pkg_resources.resource_filename(package_name, 'data/')

# use the sample dem. Change it to your own dem if needed
in_dem = os.path.join(data_dir, 'dem.tif')
# set the output directory
out_dir = os.getcwd()

# parameters for identifying sinks and delineating nested depressions
min_size = 1000      # minimum number of pixels as a depression
min_depth = 0.5      # minimum depth as a depression
interval = 0.3       # slicing interval for the level-set method
bool_shp = True      # output shapefiles for each individual level

# extracting sinks based on user-defined minimum depression size
out_dem = os.path.join(out_dir, "median.tif")
in_dem = MedianFilter(in_dem, kernel_size=3, out_file=out_dem)
sink_path = ExtractSinks(in_dem, min_size, out_dir)
dep_id_path, dep_level_path = DelineateDepressions(sink_path, 
                                                   min_size, 
                                                   min_depth, 
                                                   interval, 
                                                   out_dir, 
                                                   bool_shp)
print('Results are saved in: {}'.format(out_dir))
```

**Reply by reecejk** [reply]
Thank you for the followup. I originally utilized the ArcGIS Pro toolbox which generated the necessary maps, but it did not populate the .csv file for some reason. Maybe it had something to do with the lidar package not installed correctly, but I'm not sure how it would have generated the various .tiff files if that was the case. However, I was able to use this helpful script in the ArcGIS Pro python window to generate the .csv files I needed for my research. Thanks again!
